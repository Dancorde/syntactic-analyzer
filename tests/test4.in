program
var
const
integer
real
begin
end
read
write
procedure
if
then
else
do
while
for
to