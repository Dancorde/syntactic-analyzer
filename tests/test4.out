program - program
var - var
const - const
integer - integer
real - real
begin - begin
end - end
read - read
write - write
procedure - procedure
if - if
then - then
else - else
do - do
while - while
for - for
to - to
